lambda/write-activity-log/write-activity-log.ts

```diff
@@ -73,17 +61,17 @@ export const handler = async (event: SQSEvent): Promise<void> => {
       try {
         const activityLogEntry: ActivityLogEntry = JSON.parse(record.body);
         validateActivityLogEntry(activityLogEntry);
-        const encryptedActivities: string = await encryptData(
-          JSON.stringify(activityLogEntry.activities),
-          activityLogEntry.user_id
-        );
         const encryptedActivityLog: EncryptedActivityLogEntry = {
-          event_type: activityLogEntry.event_type,
+          event_id: activityLogEntry.event_id,
+          event_type: await encryptData(
+            activityLogEntry.event_type,
```

Comment:
@alex9smith is it correct that we only want to encrypt the event type?

Reply:
Yes that's right. Viji and I discussed with security and we agreed that was an appropriate level of defense vs. performance.